sellercenter-product-port/src/main/java/com/gfg/sellercenter/product/port/Product.java

```diff
@@ -0,0 +1,88 @@
+package com.gfg.sellercenter.product.port;
+
+import lombok.Data;
+import lombok.AllArgsConstructor;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.ZonedDateTime;
+
+@Data
+@AllArgsConstructor
+public class Product {
+    @NotNull
+    private final Integer id;
+
+    private final Integer sourceId;
+
+    private final String sku;
+
+    @NotNull
+    private final Price price;
+
+    @NotNull
+    @NotEmpty
+    private final String sellerSku;
+
+    @NotNull
+    @NotEmpty
+    private final String name;
+
+    private final Double volumetricWeight;
```

Comment:
Not null? (I don't know, but seems to be important for logistic)

Reply:
In DB it is nullable, meaning that in general, it can be omitted.